tests/opd-and-delius/opd-assessment.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { expect, test } from '@playwright/test'
+import { login as loginDelius } from '../../steps/delius/login'
+import { createOffender } from '../../steps/delius/offender/create-offender'
+import { deliusPerson } from '../../steps/delius/utils/person'
+import { login as oasysLogin, UserType } from '../../steps/oasys/login'
+import { createLayer3AssessmentWithoutNeeds } from '../../steps/oasys/layer3-assessment/create-layer3-assessment/create-layer3-without-needs'
+import { addLayer3AssessmentNeeds } from '../../steps/oasys/layer3-assessment/create-layer3-assessment/add-layer3-needs'
+import { createEvent } from '../../steps/delius/event/create-event'
+import { faker } from '@faker-js/faker'
+import * as dotenv from 'dotenv'
+import { navigateToNSIDetailsFromPersonalDetails } from '../../steps/delius/contact/find-contacts'
+dotenv.config() // read environment variables into process.env
+
+test('OPD assessment creates an event in Delius', async ({ page }) => {
+    await loginDelius(page)
+    const dob = faker.date.birthdate({ min: 20, max: 30, mode: 'age' })
+    const person = deliusPerson({ sex: 'Male', dob: dob, lastName: null, firstName: null })
```

Comment:
If you merge main into your branch, the `deliusPerson` function has been updated to take a `Partial` - which means you'd be able to omit the extra fields:
```suggestion
    const person = deliusPerson({ sex: 'Male', dob: dob })
```